lib/clockwork/test.rb

```diff
@@ -18,6 +18,11 @@ def configure(&block)
       ::Clockwork.manager.configure(&block)
       ::Clockwork::Test.manager.configure(&block)
     end
+
+    def on(event, options={}, &block)
+      ::Clockwork.manager.on(event, options, &block)
+      ::Clockwork::Test.manager.on(event, options, &block)
```

Comment:
I know you're following the convention followed throughout this file, and I apologize, because it's been a bit since I've revisiting this. Will this have any concern of running the callback twice if it's defined both on the manager and the test manager? I'm *guessing*/*hoping* no, because only one should be running at a time...but we cant verify below in the test...

Reply:
Yes, it only runs once, because `tick` runs once. I can add `.once` below.